docs/pages/extensibility/api.md

```diff
@@ -25,6 +25,29 @@ Each of the API endpoints are exposed through [server.go](https://github.com/lay
 
 Alternatively, [Remote Providers](./providers) can extend Meshery's endpoints behind the `/api/extensions/` endpoint.
 
+### How to get your token
+
+There are two ways to get your token:
+1. Meshery UI
+2. Meshery CLI
+
+- What are tokens?
+Tokens allow a user to authenticate with cloud apps and bypass two step verification and SSO, retrieve data from the instance through REST APIS
```

Comment:
nitpick: `REST APIs`

Reply:
and GraphQL APIs, too. It would be good to include a link to both of these here.